javascript/javascript1/week4/homework/voice-assistant.js

```diff
@@ -0,0 +1,163 @@
+let personName = "";
+const toDoList = [];
+const personBirthday = [];
+const monthes = [
+  "January",
+  "February",
+  "March",
+  "April",
+  "May",
+  "June",
+  "July",
+  "August",
+  "September",
+  "October",
+  "November",
+  "December",
+];
+
+function getReply(command) {
+  let commandSplit = command.split(" ");
+  let userName = commandSplit[commandSplit.length - 1];
```

Comment:
These are never reassigned, so can be `const`s.
Maybe you were thinking when the `getReply` function gets called multiple times that that would be reassigning them. But actually, because the declaration of these variables is within function, every time the function runs it creates new variables within its scope.

Check this out for more info https://dmitripavlutin.com/javascript-scope/#3-function-scope